src/libraries/System.Security.Principal.Windows/ref/System.Security.Principal.Windows.cs

```diff
@@ -20,19 +20,19 @@ namespace System.Security.Principal
     public sealed partial class IdentityNotMappedException : System.SystemException
     {
         public IdentityNotMappedException() { }
-        public IdentityNotMappedException(string message) { }
-        public IdentityNotMappedException(string message, System.Exception inner) { }
+        public IdentityNotMappedException(string? message) { }
+        public IdentityNotMappedException(string? message, System.Exception? inner) { }
         public System.Security.Principal.IdentityReferenceCollection UnmappedIdentities { get { throw null; } }
         public override void GetObjectData(System.Runtime.Serialization.SerializationInfo serializationInfo, System.Runtime.Serialization.StreamingContext streamingContext) { }
     }
     public abstract partial class IdentityReference
     {
         internal IdentityReference() { }
         public abstract string Value { get; }
-        public abstract override bool Equals(object o);
+        public abstract override bool Equals(object? o);
         public abstract override int GetHashCode();
         public abstract bool IsValidTargetType(System.Type targetType);
-        public static bool operator ==(System.Security.Principal.IdentityReference left, System.Security.Principal.IdentityReference right) { throw null; }
+        public static bool operator ==(System.Security.Principal.IdentityReference? left, System.Security.Principal.IdentityReference? right) { throw null; }
         public static bool operator !=(System.Security.Principal.IdentityReference left, System.Security.Principal.IdentityReference right) { throw null; }
```

Comment:
Shouldn't != also have nullable arguments?

Reply:
It should, thanks